pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.13
```

Comment:
We should update the version number in the [Exome Overview doc](https://broadinstitute.github.io/warp/docs/Pipelines/Exome_Germline_Single_Sample_Pipeline/README).